components/apim/init.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">= 0.15.4"
```

Comment:
use greater than 1.0.0?

Reply:
It did error with 1.0.0 due to current setting to 0.15.4, I suppose upgrading to 1.0.0 should be fine

Reply:
Resolved now @timja